src/test/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/securityrule/v4_9/CloudStackSecurityRulePluginTest.java

```diff
@@ -41,7 +42,7 @@
 @RunWith(PowerMockRunner.class)
 @PowerMockIgnore({"javax.net.ssl.*", "javax.crypto.*" })
```

Comment:
Maybe you should turn these strings into constants.

Reply:
It was removed.